modules/nw-enabling-hsts-global.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-enabling-hsts-global_{context}"]
+= Enabling HTTP strict transport security globally
+
+To enable HTTP strict transport security (HSTS) globally for all TSL routes, add a new type `IngressControllerHSTSHeaderPolicy` to the API in `openshift/api/blob/master/operator/v1/types_ingress.go` to capture the configuration of the HSTS Policy.
+
+If you have several routes or regulatory compliance requirements, you can use the global method for configuring HSTS.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Edit the Ingress conifg file:
++
+[source,terminal]
+----
+$ oc edit ingresses.config.openshift.io/cluster
+----
++
+. To enable HSTS globally for all TLS routes, set the `scope` field to `all`:
```

Comment:
This is obsolete information from the old/replaced enhancement proposal.  The new proposal is at https://github.com/openshift/enhancements/pull/849/files (with some updates that are still not merged).